### ISSUE TITLE
Paper Results Reproduction

### ISSUE THREAD

**Issue by LzqInsta** [lly00412/SEDNet#4]
hello, SEDNet is a great work which incorporates uncertainty to disparity estimation via a simple way.
Some reproduction problem has occurred when I try to use the provided pretrained model checkpoints/sceneflow/gwcnet-gc-elu-dropout-l1-logs-kl-3std-logspace-11bins-lr1e-3/checkpoint_000015.ckpt to validate SEDNet's performance on SceneFlow test split.
The metric seems to be far from the result in paper
```shell
inliers def: epe< mu+ 3sigma
inliers pct: 0.9869
+----------------------------------+
|         Global statistic         |
+--------------+----------+--------+
|     name     |   mean   |  std   |
+--------------+----------+--------+
|     EPE      |  1.1306  | 7.0258 |
|      S       |  0.453   | 1.3016 |
|      D1      | 0.045456 |   -    |
|   |EPE-S|    |  0.9887  | 6.4073 |
| |EPE-S|(mid) |  0.0429  |   -    |
+--------------+----------+--------+

```
my script to generate outputs via pretrained model is:
```shell
python main.py --dataset sceneflow \
   --datapath $DATAPATH --trainlist ./filenames/sceneflow_train.txt --testlist ./filenames/sceneflow_test.txt \
   --maxdisp 192 \
   --zoom 0.5 \
   --crop_w 448 \
   --crop_h 284 \
   --epochs 1 --lrepochs "10,12,14,16:2" \
   --batch_size 1 \
   --loss_type 'smooth_l1' \
   --save_test \
   --mask 'soft' \
   --bin_scale 'log' \
   --n_bins 11 \
   --inliers 3 \
   --model gwcnet-gcs \
   --logdir $SAVEPATH/scene_flow_model_out/ \
   --test_batch_size 2 \
   --device_id 0 \
   --loadckpt checkpoints/sceneflow/gwcnet-gc-elu-dropout-l1-logs-kl-3std-logspace-11bins-lr1e-3/checkpoint_000015.ckpt
```
and my script to compute the metrics is:
```shell
python post_process/generate_statistic.py --logdir ./checkpoints/sceneflow/scene_flow_model_out \
   --epochs 0 \
   --inliers 3 \
   --bin_scale 'log' \
   --n_bins 11 \
   --dataset 'sceneflow' \
   --uncert 
```
I don't which step is wrong.

**Comment by lly00412** [comment]
Since the sceneflow model is pre-trained in the zoom-in version, in analysis script, you also need to add:
--zoom 0.5

**Comment by LzqInsta** [comment]
thanks for your reply. I have tested with zoom=0.5
